cmd/osbuild-composer-cloud-tests/main_test.go

```diff
@@ -0,0 +1,62 @@
+// +build integration
+
+package main
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/osbuild/osbuild-composer/internal/cloudapi"
+	"github.com/stretchr/testify/require"
+	"github.com/google/uuid"
+)
+
+func TestCloud(t *testing.T) {
```

Comment:
We should probably discuss a bit how we want the tests to look in the end. I think we should have some tests that extensively verify the API (using mocked out workers to avoid building lots of images). Similarly to what is done for the weldr API. We also want functional tests that actually push images with real credentials to the cloud and boot them to verify that subscription etc works end-to-end.

Reply:
Do we have mocked workers?